fetching/strat.py

```diff
@@ -23,8 +23,12 @@ def __init__(self, overwrite_tests: bool = True, recurse_into_subdirs: bool = Tr
 
     def apply(self, project: Project):
         assert project.test_directory is not None
+        
+        file_paths = self.globber(project.test_directory, "*")
+        # Ensures that the order is deterministic.
+        sorted_file_paths = sorted(file_paths)
         for path in filter(
-            self._is_test_file, self.globber(project.test_directory, "*")
+            self._is_test_file, sorted_file_paths
```

Comment:
Why does sorting need to be done here too?

Reply:
Removed